src/components/shared/chart/Canvas.js

```diff
@@ -155,12 +169,12 @@ export default class ChartCanvas<HoveredItem> extends React.Component<
     this._offsetY = event.nativeEvent.offsetY;
     const maybeHoveredItem = this.props.hitTest(this._offsetX, this._offsetY);
 
-    // If the mouse moves too far while a button down, flag this as
-    // drag event only. Then it won't select anything when the button
-    // is released.
+    // If the mouse moves too far while the primary button is down, flag this as
+    // drag event only. Then it won't select anything when the button is
+    // released.
     if (
       !this._mouseMovedWhileClicked &&
-      event.buttons !== 0 &&
+      (event.buttons & 1) !== 0 &&
```

Comment:
I'll make it a boolean variable; but otherwise this is the only way to use `event.buttons` and this is the property to use for mousemove events. I'm not deciding about the DOM API :-)